### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by jscearce3**
## Description
Updating docs to guide end users through the registration process using the self registration plugin

## Motivation and Context

https://gitlab.jatic.net/jatic/team-metrostar/t-e-platform/-/issues/535

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,40 @@
...
+
+- Enter your email address and coupon code.
+
+- After clicking "Submit" follow the instructions to login with your temporary password. By clicking the "Login" button, it will take you to a Welcome page where you can sign in with Keycloak.
+
+<p align="center">
+  <img src="images/account-register.png" />
+</p>
+
+- After you have entered a new password, you will receive an email from "donotreply@jatic.net".  Follow the verification instructions and you will be taken to the playground.
```
Comment (kcpevey) [inline README.md:41]:
```suggestion
- After you have entered a new password, you will receive a verification email.  
```

File: README.md [hunk 2]
```diff
@@ -16,6 +16,40 @@
...
+
+<p align="center">
+  <img src="images/account-confirm.png" />
+</p>
+
+- Once your email is verified and you login you should see the Nebari landing page.
```
Comment (kcpevey) [inline README.md:47]:
```suggestion
- Once your email is verified and you login you will see the Nebari landing page.
```

**Review by kcpevey — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ericdatakelly) [reply README.md:41]: Sorry, we missed that email address.  Good catch.

**Review by ericdatakelly — APPROVED** [review]
With @kcpevey 's suggestions, it looks good to merge.

**Comment by kenafoster** [comment]
The guidance is still very JATIC-specific, but since this is in the README for the general extension, I think any references to JATIC, CDAO, or playground need to be wiped out.  I'll request some specific changes

It is going to be tricky to get the right screenshots for "general" until the issue for generic/default branding of the plugin is completed.  Could just make some in-browser edits to get close enough I think.

File: images/main-landing.png
Comment (kenafoster) [inline images/main-landing.png:1]: Screenshot should not reference JATIC or Nebari.  Note all the content in the screenshot is configurable in nebari-config.  Attached is a version that uses the default "welcome" text

image

File: images/nebari-landing.png
Comment (kenafoster) [inline images/nebari-landing.png:1]: Screenshot that strips out the customer-specific stuff

image

**Review by kenafoster — CHANGES_REQUESTED** [review]

**Review by kenafoster — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -16,6 +16,36 @@
...
 
+## User Registration via this extension
+Steps for self registration:
+
+- Navigate to your Nebari domain.
+<p align="center">
+  <img src="images/welcome-nebari.png" />
+</p>
+
+- Click the link at "To register for an account, click here." or go to https://{your-domain-name}/registration
```
Comment (kcpevey) [inline README.md:27]: This doesn't match the image above. The image needs to have this text.

**Review by kcpevey — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (jscearce3) [reply README.md]: Good catch, the image that had that hyper link had JATIC specific language we were trying to remove. I changed it up a bit, but maybe it just makes the most sense for step 1 to be "Navigate to https://{your-domain-here}/registration"

**Review by kcpevey — APPROVED** [review]
LGTM! Thanks @jscearce3 !

**Review by tylergraff — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (tylergraff) [reply README.md]: Resolved by: https://github.com/MetroStar/nebari-self-registration/pull/9/commits/05fba93705caaef8d617b2635558592a349c5951

**Pull request merged**